DuckDuckGo/AdAttribution/AdAttributionPixelReporter.swift

```diff
@@ -24,7 +24,7 @@ protocol PixelFiring {
     static func fire(pixel: Pixel.Event, withAdditionalParameters params: [String: String], includedParameters: [Pixel.QueryParameters]) async throws
 }
 
-final class AdAttributionPixelReporter {
+final actor AdAttributionPixelReporter {
```

Comment:
Changing this to an actor so that multiple detached Tasks trying to report attribution at once will behave as we'd like.